### PR TITLE
[IDL] Add parameter to GENERATE_IDL_PYTHON

### PR DESCRIPTION
**Pull request by jmirabel**

### CODE REVIEW

File: idl.cmake [hunk 1]
```diff
@@ -135,8 +135,14 @@ ENDMACRO(GENERATE_IDL_CPP FILENAME DIRECTORY)
 #                    Can be prefixed by a path: _path/_filename
 #   :param DIRECTORY: IDL directory.
 #                     The idl file being search for is: ``${DIRECTORY}/${_filename}.idl``
+#   :param ARGUMENTS:  The following words are passed as arguments to omniidl
 #
 MACRO(GENERATE_IDL_PYTHON FILENAME DIRECTORY)
+  SET(options)
+  SET(oneValueArgs )
+  SET(multiValueArgs ARGUMENTS)
```
Comment (jcarpent) [inline idl.cmake]: Where the `ARGUMENTS` comes from here?

Comment (jmirabel) [reply idl.cmake]: It tells CMake to parse this in the remaining arguments, i.e. `foo (a b ARGUMENTS c d)` leads to `_omni_ARGUMENTS` contain `c;d`

Comment (jcarpent) [reply idl.cmake]: OK, thanks for this clarification.

**Comment by jcarpent** [comment]
I will merge it now.

**Comment by jmirabel** [comment]
Thanks.

**Pull request merged**